client/app/services/event-notifications.service.js

```diff
@@ -42,21 +42,73 @@
       }
     };
 
+    function miqFormatNotification(text, bindings) {
+      var str = __(text);
+      lodash.each(bindings, function (value, key) {
+        str = str.replace(new RegExp('%{' + key + '}', 'g'), value.text);
+      });
+
+      return str;
+    }
+
     doReset();
 
+    var cable = ActionCable.createConsumer('/ws/notifications');
+
+    cable.subscriptions.create('NotificationChannel', {
+      disconnected: function () {
+        // eslint-disable-next-line consistent-this
```

Comment:
Oh yeah, this one we should look into.. `vm` makes sense as an alias for `this` *only* in the context of angular controllers, where it's a convention that stands for view-model.  But there's a lot of places in JS where you need to save this somewhere else (at least until we move to ES6) - so I'd be for relaxing the rule, or, if we want to be strict, extending the list of options to something like `"vm", "self"` or `"vm", "service", ...`..